### PR TITLE
CODEOWNERS: installer, zephyr and tools/topology

### PR DESCRIPTION
**Pull request by kv2019i**

### CODE REVIEW

**Comment by kv2019i** [comment]
@ranj063 Noted you are not getting added to all topology2 reviews. Added @jsarha you as well. And while at it, a few additionas that have not been covered by anyone so far. More volunteers are welcome are btw! This just based on who has been reviewing these folders in recent history.

File: CODEOWNERS [hunk 1]
```diff
@@ -70,7 +73,8 @@ scripts/*				@marc-hb @aborisovich
 
 # tools(old 'soft' repo)
 tools/logger/*				@bkokoszx @akloniex
-tools/topology/*			@ranj063
+tools/topology/**			@ranj063
+tools/topology/topology2/**		@jsarha
```
Comment (marc-hb) [inline CODEOWNERS:77]: Last pattern wins, so a PR that touches `tools/topology/topology2/foo.conf` only will not add @ranj063. Is that intended?

File: CODEOWNERS [hunk 2]
```diff
@@ -10,6 +10,9 @@
 # File patterns work mostly like .gitignore. Try to keep this file
```
Comment (marc-hb) [inline CODEOWNERS:10]: Off-topic sorry but can you please drop the "javascript" comment above and replace it with a link to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners?

It looks like copy/pasta from there.

This feature is too complex, it's badly missing a test script like
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/scripts/get_maintainer.pl

File: CODEOWNERS [hunk 3]
```diff
@@ -10,6 +10,9 @@
...
 # simple because it's literally impossible to test.
 
+# installer
+installer/**				@marc-hb
```
Comment (marc-hb) [inline CODEOWNERS:14]: OK but can you move it closer to the CMakeLists.txt rules below? Just before them I guess.

**Comment by iuliana-prodan** [comment]
> @dbaluta @paulstelian97 maybe one of you should be listed for zephyr/ as well ...?

@kv2019i please add myself and @dbaluta for zephyr/. Thanks!

File: CODEOWNERS (hunk 1 above)
Comment (kv2019i) [reply CODEOWNERS]: Good catch, thanks @marc-hb . Fixed in latest push.

File: CODEOWNERS (hunk 2 above)
Comment (kv2019i) [reply CODEOWNERS]: Ack, added @marc-hb

File: CODEOWNERS (hunk 3 above)
Comment (kv2019i) [reply CODEOWNERS]: ALso done.

**Comment by kv2019i** [comment]
@iuliana-prodan wrote:
> > @dbaluta @paulstelian97 maybe one of you should be listed for zephyr/ as well ...?
> 
> @kv2019i please add myself and @dbaluta for zephyr/. Thanks!

Sure thing, now added in updated push.

**Review by lgirdwood — APPROVED** [review]
More codeowner updates coming from @mwasko soon. This will all help with review.

File: CODEOWNERS [hunk 4]
```diff
@@ -7,8 +7,8 @@
 # Order is important. The last matching pattern has the most precedence.
 # So if a pull request only touches javascript files, only these owners
 # will be requested to review.
-# File patterns work mostly like .gitignore. Try to keep this file
-# simple because it's literally impossible to test.
+#
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
```
Comment (marc-hb) [inline CODEOWNERS:11]: Ouch that's not the bit I asked to remove... will submit a follow-up PR.

**Pull request merged**